mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -88,6 +90,35 @@ See [Enabling CMEK for Logging Buckets](https://cloud.google.com/logging/docs/ro
 			},
 		},
 	},
+	"index_configs": &schema.Schema{
+		Type:        schema.TypeSet,
+		MaxItems:    20,
+		Optional:    true,
+		Description: `A list of indexed fields and related configuration data.`,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"field_path": &schema.Schema{
+					Type:        schema.TypeString,
+					Required: true,
+					Description: `The LogEntry field path to index.`,
+				},
+				"type": &schema.Schema{
+					Type:     schema.TypeString,
+					Required: true,
+					Description: `The type of data in this index
+Note that some paths are automatically indexed, and other paths are not eligible for indexing. See [indexing documentation]( https://cloud.google.com/logging/docs/view/advanced-queries#indexed-fields) for details.
+For example: jsonPayload.request.status`,
+				},
+				"create_time": &schema.Schema{
```

Comment:
We should leave this field off if not absolutely necessary `schema.TypeSet` does not play well with `Computed` fields, due to how diffs are calculated for sets, and Terraform tries to reuse old computed values.